autogluon/task/tabular_prediction/predictor.py

```diff
@@ -624,6 +627,77 @@ def info(self):
         """
         return self._learner.get_info(include_model_info=True)
 
+    # TODO: Handle cases where name is same as a previously fit model, currently overwrites old model.
+    # TODO: Add dataset argument
+    # TODO: Add option to disable OOF generation of newly fitted models
+    # TODO: Move code logic to learner/trainer
+    # TODO: Add task.fit arg to perform this automatically at end of training
+    def fit_weighted_ensemble(self, base_models: list = None, name_suffix='_custom', expand_pareto_frontier=False, time_limits=None):
```

Comment:
Why not keep this method hidden inside of Learner until it's no longer experimental? 
Or name it `_ fit_weighted_ensemble`?

Reply:
In the future, we could consider adding args like:
`topmost_accurate_cutoff`, 
`fastest_cutoff`.

Where if fastest_cutoff = k (and base_models not specified), then base_models is set equal to the k fastest models (similarly for topmost_accurate_cutoff = k implies base_models is set to the k most-accurate models)

Reply:
Removed experimental warning, added future TODOs